src/coins.h

```diff
@@ -750,24 +741,19 @@ class CCoinsViewCache : public CCoinsViewBacked
     friend class CCoinsModifier;
 
 private:
-    CCoinsMap::const_iterator      FetchCoins(const uint256 &txid)       const;
-    CCoinsMap::iterator            FetchCoins(const uint256 &txid);
-    CSidechainsMap::const_iterator FetchSidechains(const uint256& scId)  const;
-    CSidechainsMap::iterator       ModifySidechain(const uint256& scId);
-    const CSidechain* const        AccessSidechain(const uint256& scId);
-    CSidechainEventsMap::const_iterator FetchSidechainEvents(int height) const;
-    CSidechainEventsMap::iterator  ModifySidechainEvents(int height);
+    CCoinsMap::const_iterator           FetchCoins(const uint256 &txid)       const;
+    CCoinsMap::iterator                 FetchCoins(const uint256 &txid);
+    CSidechainsMap::const_iterator      FetchSidechains(const uint256& scId)  const;
+    CSidechainsMap::iterator            ModifySidechain(const uint256& scId);
+    const CSidechain* const             AccessSidechain(const uint256& scId);
+    CSidechainEventsMap::const_iterator FetchSidechainEvents(int height)      const;
+    CSidechainEventsMap::iterator       ModifySidechainEvents(int height);
+    CCertDataHashMap::const_iterator    FetchCertDataEntry(const uint256& scId, const int epoch) const;
```

Comment:
Add `ModifyCertDataEntry` to make CerDataEntries modification similar to other approaches (ModifySidechainEvents, ModifySidechain).

Reply:
Done